google/cloud/bigquery/magics.py

```diff
@@ -66,7 +66,10 @@
           the variable name (ex. ``$my_dict_var``). See ``In[6]`` and ``In[7]``
           in the Examples section below.
     * ``<query>`` (required, cell argument):
-        SQL query to run.
+        SQL query to run. If the query does not contain any whitespace (aside
+        from leading and trailing whitespace), it is assumed to represent a
+        fully-qualified table ID, and the latter's data will be fetched and
```

Comment:
nit:  possibly remove the extra reference in the docstring to returning dataframe here?  As written I expected this to mean that this special case returned something different.

Reply:
Sure, why not, let's remove this possible source of confusion then.